src/server/game/Movement/MovementGenerators/PointMovementGenerator.cpp

```diff
@@ -75,13 +75,13 @@ bool PointMovementGenerator<T>::DoUpdate(T* owner, uint32 /*diff*/)
         return true;
     }
 
-    owner->AddUnitState(UNIT_STATE_ROAMING_MOVE);
-
     if ((_interrupt && owner->movespline->Finalized()) || (_recalculateSpeed && !owner->movespline->Finalized()))
     {
         _recalculateSpeed = false;
         _interrupt = false;
 
+        owner->AddUnitState(UNIT_STATE_ROAMING_MOVE);
```

Comment:
Thoses changes concerned the PointMovementGenerator but were in the Targeted MoveGen commit. Ccrs already fixed it though.